EnergyPlus_Engine/Compute/SimulateIDF.cs

```diff
@@ -27,17 +27,18 @@
 using System.ComponentModel;
 using System.IO;
 
-namespace BH.Engine.Radiance
+namespace BH.Engine.EnergyPlus
 {
     public static partial class Compute
     {
         [Description("Run an IDF file")]
         [Input("energyPlusSettings", "EnergyPlusSettings object")]
         [Input("idfFile", "EnergyPlus IDF file to simulate")]
+        [Input("run", "Set to \"True\" to run simulation")]
         [Output("success", "True if command has been run sucessfully (subject to program being called correctly returning an Exit Code)")]
-        public static bool SimulateIDF(EnergyPlusSettings energyPlusSettings, string idfFile)
+        public static bool SimulateIDF(EnergyPlusSettings energyPlusSettings, string idfFile, bool run = false)
         {
-            string formatString = "{0} -a -r -x -d {1} -p {2} -w {3} {4}";
+            string formatString = "{0} -r -x -d {1} -p {2} -w {3} {4}";
```

Comment:
I suspect it might be useful to make use of the `run` argument somewhere? I'd add

```
if(!run)
    return false;
```

Above this line?

Reply:
Haha, that's a bit of an omission isn't it! That's been dealt with now.

On the subject of running executables with arguments, I'm wondering if there's a [better way to handle this](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.arguments?view=netcore-3.1) as it's becoming more common. A discussion for another place maybe?